src/relay/backend/vm/compiler.cc

```diff
@@ -780,31 +780,81 @@ PackedFunc VMCompiler::GetFunction(const std::string& name,
   if (name == "compile") {
     return PackedFunc([sptr_to_self, this](TVMArgs args, TVMRetValue* rv) {
       CHECK_EQ(args.num_args, 3);
-      this->Compile(args[0], args[1], args[2]);
+      Module mod = args[0];
+      this->Compile(mod, args[1], args[2]);
     });
   } else if (name == "get_vm") {
     return PackedFunc([sptr_to_self, this](TVMArgs args, TVMRetValue* rv) {
       *rv = runtime::Module(vm_);
     });
+  } else if (name == "set_params") {
```

Comment:
Since we add `set_params` in compiler, we should also add `get_params` function. 
I assume that the AlterLayout+FoldConstant will change the layout of params. Therefore, we should return params in the python compile function.

Reply:
thanks for the reminder. added `get_params`